tests/functional/ext/typing/redundant_typehint_argument.py

```diff
@@ -0,0 +1,21 @@
+""""Checks for redundant Union typehints in assignments"""
+# pylint: disable=deprecated-typing-alias,consider-alternative-union-syntax,consider-using-alias
+
+from __future__ import annotations
+from typing import Union, Optional, Sequence
+
+#  +1: [redundant-typehint-argument, redundant-typehint-argument]
+ANSWER_0: Union[int, int, str, bool, float, str] = 0  #
```

Comment:
```suggestion
ANSWER_0: Union[int, int, str, bool, float, str] = 0
```

Reply:
@Pierre-Sassoulas 
I thought the extra `#` was needed..
Maybe the docs need a refactor, or I just didn't get it quite right...

from:
https://pylint.pycqa.org/en/latest/development_guide/contributor_guide/tests/writing_test.html

```
You can also use # +n: [ with n an integer if the above syntax would make the line too long or other reasons:

# +1: [empty-comment]
#
```

Reply:
Ha I understand. The additional ``#`` is just to signal that there is a line under it I think. Maybe we should add a comment "There error is actually here" or something ?

Reply:
Oh my god! now I get it.  The second comment is the `empty-comment` error. Therefore the second line `#` is not a part of the syntax it is just the line itself that has the error.
It is very unclear... maybe we should change the example to something clearer like:


```
A = 5
# +1: [singleton-comparison]
B = A == None
```

Reply:
opened a separate PR to make this mystery clearer for future contributors.
#7870

Reply:
> The second comment is the empty-comment error. 

Now I remember this 😄 . There's clearly no other way to do a functional test for ``[empty-comment]`` 😄 Thank you for making the doc clearer 👍